data/data-pipeline/data_pipeline/etl/sources/mdejscreen/etl.py

```diff
@@ -0,0 +1,112 @@
+
+        # Load comparison shape files that comprise Maryland EJScreen
+        # https://p1.cgis.umd.edu/mdejscreen/help.html
+        # From the documentation:
+        # Each of the following categories contains a number of relevant indicators,
+        # and an averaged score. The two "Pollution Burden" average scores are then
+        # averaged together and the result is multiplied by the average of the
+        # "Population Characteristics" categories to get the total EJ Score of each tract.
+        # For each indicator, the percentile is given.
+        # For example, the indicator value for "Asthma Emergency Discharges" with 0.9 is therefore in
+        # the 90th percentile, which means only 10% of tracts in Maryland have higher values.
+        # EJ Scores near 1 represent areas of the greatest environmental justice concern.
+
+        list_of_files = list(glob(str(self.SHAPE_FILES_PATH) + "/*.shp"))
+        # # ignore counties becauses this is not the level of measurement
+        # # that is consistent with our current scoring and ranking methodology
+
+        dfs_list = [
+            gpd.read_file(f)
+            for f in list_of_files
+            if not f.endswith("CountiesEJScore.shp")
+        ]
+
+        # set the Census tract as the index and drop the geometry column
+        # that produces the census tract boundaries
+        # geopandas raises an exception if there are duplicate geometry columns
+        # Moreover, since the unit of measurement is at the tract level
+        # we can consistantly merge this with another dataset
+        dfs_list = [
+            df.set_index("Census_Tra").drop("geometry", axis=1)
+            for df in dfs_list
+        ]
+        # pylint: disable=unsubscriptable-object
+        combined_df = gpd.GeoDataFrame(pd.concat(dfs_list, axis=1))
+
+        # reset index so that we no longer have the tract as our index
+        combined_df = combined_df.reset_index()
+        # coerce into integer into
+        # pylint: disable=unsupported-assignment-operation, unsubscriptable-object
+        combined_df["Census_Tra"] = (combined_df["Census_Tra"]).astype(int)
+
+        # drop the 10 census tracts that are zero: please see here:
+        # https://github.com/usds/justice40-tool/issues/239#issuecomment-995821572
+        combined_df = combined_df[combined_df["Census_Tra"] != 0]
+
+        self.df = combined_df.copy()
+
+        self.df.rename(
+            columns={
+                "Census_Tra": self.GEOID_TRACT_FIELD_NAME,
+                "EJScore": self.MDEJSCREEN_PERCENTILE_FIELD_NAME,
+            },
+            inplace=True,
+        )
+
+        # Baseline Comparisons at quartiles and the 90th percentile
+        # Interpretation: The score is greater than
+        # N% of the tracts in the state
+        self.df[field_names.MDEjSCREEN_TRACT_25_PERCENT_FIELD] = (
+            self.df[self.MDEJSCREEN_PERCENTILE_FIELD_NAME] > 0.25
```

Comment:
I added these for future analyses and for comparisons with other measures used in Census tract datasets.

Reply:
Does MD EJSCREEN team never use a particular cutoff anywhere for this? Is there a reason you picked 25% here?

Reply:
I will re-review the associated papers and the original white paper again, and reconfirm using the associated [paper](https://www.mdpi.com/1660-4601/16/3/348/htm) that we had discussed earlier this month. 

On cursory glance with an example - citing Figure 4 from the aforementioned paper (below) - there appears to be an interval of values that are produced (albeit from the scoring perspective):

![image](https://user-images.githubusercontent.com/94847739/147761267-be6dd081-40f1-4924-9177-edce341a6cbf.png)

Again, I will re-review the reference and the associated references to double check. In addition, is it all possible for us to receive feedback from Dr. Wilson on a reasonable threshold for burden (assuming the literature review doesn't produce anything definitive)?